.storybook/stories/AuthenticatedAppShell/AuthenticatedAppShell.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Flex } from '@ag.ds-next/react/box';
+import { tokens, useTernaryState } from '@ag.ds-next/react/core';
+import { ReactNode } from 'react';
+import { AuthenticatedAppShellHeader } from './AuthenticatedAppShellHeader';
+import {
+	AuthenticatedAppShellNav,
+	AuthenticatedAppShellNavListItemType,
+} from './AuthenticatedAppShellNav';
+import { AuthenticatedAppShellFooter } from './AuthenticatedAppShellFooter';
+
+export const AuthenticatedAppShell = ({
+	siteTitle,
+	siteSubtitle,
+	children,
+	collapseByDefault = false,
+	navItems,
+}: {
+	siteTitle: string;
+	siteSubtitle: string;
+	children: ReactNode;
+	collapseByDefault?: boolean;
+	navItems: AuthenticatedAppShellNavListItemType[];
+	signOut: () => void;
+}) => {
+	const [isOpen, openMenu, closeMenu] = useTernaryState(!collapseByDefault);
```

Comment:
I think this state should be managed by the consumer (so move it app one level). That way, they can store the state in either local storage, or potentially even in a database.